lib/platform/gitea/index.ts

```diff
@@ -633,6 +634,11 @@ const platform: Platform = {
       if (!issues.length) {
         issues = issueList.filter((i) => i.title === reuseTitle);
       }
+
+      const labels = Array.isArray(labelNames)
+        ? await Promise.all(labelNames.map(lookupLabelByName))
```

Comment:
Use `p-map` to limit to 
```suggestion
        ? await pMap(labelNames, lookupLabelByName, { concurrency: 5 })`

Reply:
I'm not sure whether concurrency is what you want here, as the method called (`lookupLabelByName`) uses an caching mechanism in order to not retrieve al labels on each invocation...

Reply:
And actually, this is almost literally copied from a few lines earlier https://github.com/renovatebot/renovate/blob/49072b970edee5b6e83f3c6802b6d8ee0da4b150/lib/platform/gitea/index.ts#L490

Reply:
ok, i see. ignore this change.